services/s3/src/it/java/software/amazon/awssdk/services/s3/multipart/S3ClientMultiPartCopyIntegrationTest.java

```diff
@@ -166,6 +184,8 @@ private void createOriginalObject(byte[] originalContent, String originalKey) {
         s3CrtAsyncClient.putObject(r -> r.bucket(BUCKET)
                                          .key(originalKey),
                                    AsyncRequestBody.fromBytes(originalContent)).join();
+
+        CAPTURING_INTERCEPTOR.reset();
```

Comment:
Do you need to reset the interceptor here? seems like the value you're checking is set after anyway

Reply:
it will be set for PUT, so reset to check for COPY. 
looks like CRT doesn't use CRC32 for COPY, only checking in single test with CRC32 set explicitly